FHIR/FSHOnline
# Update dependencies

**Description:**
Packages that remain out of date are explained in dependency notes. Of particular note is `eslint-plugin-flowtype`, which at time of writing has an [open issue](https://github.com/gajus/eslint-plugin-flowtype/issues/528) to release an updated version with eslint v9 support.

**Testing Instructions:**
Confirm that all tests pass locally, and that linting works as expected. Confirm that `npm audit` reports no vulnerabilities. Confirm that `npm outdated` reports only the packages that are listed in `DEPENDENCY-NOTES.md`. Confirm that you are able to build and run application locally.

**Related Issue:**
N/A
